tests/frontend/unit/FirebaseApiClient.test.js

```
@@ -0,0 +1,483 @@
+      
+      // Apply filters if there are any
+      if (conditions.length > 0) {
+        items = items.filter(item => {
+          return conditions.every(condition => {
+            const [field, op, value] = condition;
+            
+            if (op === '==') {
+              return item[field] === value;
+            } else if (op === '!=') {
+              return item[field] !== value;
+            }
```

Biome (1.9.4)

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)